swagger.json

```diff
@@ -40,9 +40,202 @@
         {
             "name": "Default",
             "description": "All Samsara API endpoints"
+        },
+        {
+            "name": "Users",
+            "description": "Access to user data"
         }
     ],
     "paths": {
+        "/users": {
+            "get": {
+                "tags": [
+                    "Default", "Users"
+                ],
+                "summary": "/users",
+                "description": "Get all users in the organization.",
+                "operationId": "list_users",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "List of users.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/User"
+                            }
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "tags": [
+                    "Default", "Users"
+                ],
+                "summary": "/users",
+                "description": "Add a user to the organization.",
+                "operationId": "create_user",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    { "$ref": "#/parameters/createUserParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "User successfully created.",
+                        "schema": {
+                            "$ref": "#/definitions/User"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/users/{user_id}": {
+            "get": {
+                "tags": [
+                    "Default", "Users"
+                ],
+                "summary": "/users/{user_id:[0-9]+}",
+                "description": "Get a user.",
+                "operationId": "get_user",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" },
+                    {
+                        "name": "user_id",
```

Comment:
Nit: might make sense to move this to `#/parameters/userId` and share it with the other /users endpoints.